modules/ROOT/pages/composite-databases/administration.adoc

```diff
@@ -1,102 +0,0 @@
-:description: This section describes the administration and operation of Composite databases.
-[role=enterprise-edition]
-[[composite-databases-administration]]
-= Managing Composite databases
-
-Composite databases are managed using administrative commands.
-They are created with the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/databases#administration-databases-create-composite-database[`CREATE COMPOSITE DATABASE`^] command.
-
-== Create a Composite database
-
-====
-[source, cypher]
-----
-CREATE COMPOSITE DATABASE cineasts
-----
-====
-
-Constituent graphs are added with the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/aliases#alias-management-create-database-alias[`CREATE ALIAS`^] administrative command, for example:
-
-== Create an alias on a Composite database
-
-====
-[source, cypher]
-----
-CREATE ALIAS `cineasts`.`latest`
-  FOR DATABASE movies2022
-----
-====
-
-[NOTE]
-====
-`.` is used as a separating character between the namespace and database name in aliases.
-It is therefore recommended to avoid `.` characters in database and composite database names.
-If '.' is needed in either name, the name must be surrounded by backticks, i.e. `composite.`db.name``.
-Backticks may be added regardless of whether the name contains special characters or not, so it is good practice to always backtick both names, i.e. `pass:[`composite`.`db.name`]`.
-For more details on escaping special characters, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/aliases/#alias-management-escaping[Alias names and escaping].
-====
```

Comment:
More a reminder for me to remember to check this when I get to the relevant file than a review comment: Does this note still live anywhere? it seems like a useful note to have

Reply:
While the note in its fullness doesn't seem to exist anymore most of the information exists in other notes in relevant places. I think the only thing I'm missing is
```
Backticks may be added regardless of whether the name contains special characters or not, so it is good practice to always backtick both names, i.e. `pass:[`composite`.`db.name`]`.
```
which I don't think was mentioned anywhere? (maybe `Alias names and escaping` is a good place?)